### PR TITLE
[webview_flutter] add scrollViewBounces settings

### PR DESCRIPTION
**Pull request by yinsee**
## Description

*add scrollViewBounces settings to disable bounce effect on iOS.*
https://github.com/flutter/flutter/issues/57090

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [x] All existing and new tests are passing.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] The analyzer (`flutter analyze`) does not report any problems on my PR.
- [x] I read and followed the [Flutter Style Guide].
- [x] The title of the PR starts with the name of the plugin surrounded by square brackets, e.g. [shared_preferences]
- [ ] I updated pubspec.yaml with an appropriate new version according to the [pub versioning philosophy].
- [ ] I updated CHANGELOG.md to add a description of the change.
- [x] I signed the [CLA].
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (please indicate a breaking change in CHANGELOG.md and increment major revision).
- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/flutter/plugins/blob/master/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://www.dartlang.org/tools/pub/versioning
[CLA]: https://cla.developers.google.com/

### CODE REVIEW

**Comment by yinsee** [comment]
@googlebot I signed it!

**Comment by ened** [comment]
@yinsee this may need a test in `webview_flutter_test.dart`, just like `debuggingEnabled` has.

**Comment by yinsee** [comment]
@ened added test. thanks!

**Comment by ened** [comment]
@yinsee could you try restarting the failed checks (click on Details -> retry, I think) so that we have it green?

File: packages/webview_flutter/lib/webview_flutter.dart [hunk 1]
```diff
@@ -278,6 +279,11 @@ class WebView extends StatefulWidget {
   /// By default `debuggingEnabled` is false.
   final bool debuggingEnabled;
 
+  /// Controls whether WebView have bounce effect in iOS
+  ///
+  /// By default `scrollViewBounces` is true.
+  final bool scrollViewBounces;
```
Comment (amirh) [inline packages/webview_flutter/lib/webview_flutter.dart]: I'm wondering whether it makes sense to phrase this as a platform-agnostic API?
e.g I'd imagine that on Android an equivalent knob will control whether the overflow glow is showing or not (not sure whether the Android webview currently allows such control).

In that case maybe it makes sense to name this something like `allowOverScroll` ?

**Comment by jamesdixon** [comment]
@amirh any update on this review?

cc: @yinsee

**Comment by stuartmorgan** [comment]
Thank you for your contribution. I'm going to close this PR for now since there are outstanding comments, just to get this off our PR review queue. Please don't hesitate to submit a new PR if you have the time to address the review comments. Thanks!

**Pull request closed without merging**